scaling.py

```diff
@@ -25,8 +28,8 @@ def fscalling(X_train, X_test, patchSize):
         zaxis_train = np.arange(lenTrain)
         zaxis_test = np.arange(lenTest)
 
-        ############ in batches
-        ########### Up scaling the training set
+        # in batches
+        # Up scaling the training set
         Batch=20
```

Comment:
Why batch = 20?
Load it from config file

Reply:
still hard-coded batch size? load from config

Reply:
This batch parameter is just for the processing of scaling functions.It just helps the function work faster and has no influence on the result or patches. Can I keep it here?

Reply:
in that case you may keep it there